core/persist/persistenceManager/Types.ts

```diff
@@ -0,0 +1,34 @@
+import {PlainObject} from '@xh/hoist/core';
+import {LocalDate} from '@xh/hoist/utils/datetime';
+
+export interface PersistenceView<T extends PlainObject = PlainObject> {
+    id: number;
```

Comment:
I know this is from the JsonBlob interface, but wondering for my own knowledge - why the need for id and token if token is the unique ID for the blob?